.doc_gen/metadata/route53_metadata.yaml

```diff
@@ -0,0 +1,35 @@
+# zexi 0.4.3
+route-53_GetHostedZoneCount:
+  title: Get the number of hosted zones that are associated with the current &AWS; account using an &AWS; SDK
+  title_abbrev: Get the number of hosted zones
+  synopsis: get the number of hosted zones.
+  category:
+  languages:
+    Rust:
+      versions:
+        - sdk_version: 1
+          github: rust_dev_preview/route53
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - route53.rust.route53-helloworld
```

Comment:
This snippet doesn't exist.

Reply:
Strange. I must have forgotten to check in the updated code example. Done.